Xamarin.Essentials/Types/Placemark.shared.cs

```diff
@@ -30,34 +31,53 @@ public Placemark(Placemark placemark)
             AdminArea = placemark.AdminArea;
         }
 
-        public Location Location { get; set; }
+        public Location? Location { get; set; }
 
-        public string CountryCode { get; set; }
+        public string? CountryCode { get; set; }
 
-        public string CountryName { get; set; }
+        public string? CountryName { get; set; }
 
-        public string FeatureName { get; set; }
+        public string? FeatureName { get; set; }
 
-        public string PostalCode { get; set; }
+        public string? PostalCode { get; set; }
 
-        public string SubLocality { get; set; }
+        public string? SubLocality { get; set; }
 
-        public string Thoroughfare { get; set; }
+        public string? Thoroughfare { get; set; }
 
-        public string SubThoroughfare { get; set; }
+        public string? SubThoroughfare { get; set; }
 
-        public string Locality { get; set; }
+        public string? Locality { get; set; }
 
-        public string AdminArea { get; set; }
+        public string? AdminArea { get; set; }
 
-        public string SubAdminArea { get; set; }
+        public string? SubAdminArea { get; set; }
 
-        public override string ToString() =>
-            $"{nameof(Location)}: {Location}, {nameof(CountryCode)}: {CountryCode}, " +
-            $"{nameof(CountryName)}: {CountryName}, {nameof(FeatureName)}: {FeatureName}, " +
-            $"{nameof(PostalCode)}: {PostalCode}, {nameof(SubLocality)}: {SubLocality}, " +
-            $"{nameof(Thoroughfare)}: {Thoroughfare}, {nameof(SubThoroughfare)}: {SubThoroughfare}, " +
-            $"{nameof(Locality)}: {Locality}, {nameof(AdminArea)}: {AdminArea}, " +
-            $"{nameof(SubAdminArea)}: {SubAdminArea}";
+        public override string ToString()
+        {
+            var sb = new StringBuilder();
```

Comment:
replaced huge strinig format+concat with stringbuilder. StringJoin is not available sadly